internal/statemachine/helper.go

```diff
@@ -812,7 +812,7 @@ func manualAddGroup(addGroupInterfaces interface{}, targetDir string, debug bool
 	addGroupSlice := reflect.ValueOf(addGroupInterfaces)
 	for i := 0; i < addGroupSlice.Len(); i++ {
 		addGroup := addGroupSlice.Index(i).Interface().(*imagedefinition.AddGroup)
-		addGroupCmd := execCommand("chroot", targetDir, "addgroup", addGroup.GroupName)
+		addGroupCmd := execCommand("chroot", targetDir, "groupadd", addGroup.GroupName)
```

Comment:
This (and the other change to this file) seems unrelated?

Reply:
They are, yes. There was a change to debootstrap that no longer includes the `useradd` package, which was causing these tests to fail. If you'd prefer I can make this a separate PR.

Reply:
I would like these changes to be in a commit that explains them but I don't think they need to be in a separate PR

Reply:
I can do that.